package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fulcrum-core",
-  "version": "0.14.2",
+  "version": "0.14.3",
```

Comment:
Seems like it is a new feature, not a bugfix. So `0.15.0`.

Reply:
whoops fixing